schemas/config.json

```diff
@@ -0,0 +1,397 @@
+                            "enum": [
+                                "update",
+                                "recreate",
+                                "fail",
+                                "ask"
+                            ]
+                        },
+                        "templates_folder": {
+                            "title": "Templates folder",
+                            "description": "Folder where source generation jinja templates are located. Override default templates creating  source_props.yml, source_model_props.yml, and source_model.sql under this folder",
+                            "type": "string",
+                            "minLength": 1
+                        },
+                        "metadata": {
+                            "title": "Metadata",
+                            "description": "",
+                            "type": "string",
+                            "minLength": 1
+                        }
+                    },
+                    "required": [],
+                    "dependencies": {
+                        "relations": [
+                            "database",
+                            "schemas"
+                        ],
+                        "schemas": [
+                            "database"
+                        ]
+                    }
+                },
+                "properties": {
+                    "title": "Properties",
+                    "description": "Generate properties parameters",
+                    "type": "object",
+                    "properties": {
+                        "templates_folder": {
+                            "title": "Templates folder",
+                            "description": "",
+                            "type": "string",
+                            "minLength": 1
+                        },
+                        "metadata": {
+                            "title": "Metadata",
+                            "description": "",
+                            "type": "string",
+                            "minLength": 1
+                        },
+                        "update-strategy": {
+                            "title": "Update strategy",
+                            "description": "Action to perform when a property file already exists. Options: update, recreate, fail, ask (per file)",
+                            "type": "string",
+                            "enum": [
+                                "update",
+                                "recreate",
+                                "fail",
+                                "ask"
+                            ]
+                        },
+                        "destination": {
+                            "title": "Destination",
+                            "description": "Where models yml files will be generated\nYou can specify a different path by declaring it explicitly, i.e.: \"models/staging/{{model_file_name}}.yml\"",
```

Comment:
Default location and file name where model yml files will be generated. You can specify a specific path by declaring it explicitly, i.e.: \"models/staging/{{model_file_name}}.yml\" or \"{{model_folder_path}}/{{model_file_name}}.yml\"